cli/flags.rs

```diff
@@ -701,6 +706,14 @@ Ignore formatting a file by adding an ignore comment at the top of the file:
         .multiple(true)
         .required(false),
     )
+    .arg(
+      Arg::with_name("verbose")
+        .long("verbose")
+        .short("v")
+        .requires("unstable")
```

Comment:
I wonder if it's necessary to mark `verbose` flag as unstable in `deno fmt`

Reply:
I think we can skip `unstable` in this case

Reply:
Done.